docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -119,10 +119,10 @@ You can create a new tag, which is not in the default list, by following the pro
 
 .Procedure
 
-. In *Developer* view, click *Controls*.
+. In *Migration* view, click *Controls*.
 . Click *Tags*.
 . Click *Create tag*.
 . Enter the following information:
 ** Name
-** Tag type
+** Category
```

Comment:
can you please replace with " **Tag category** "

Reply:
Done